.circleci/config.yml

```diff
@@ -26,7 +26,10 @@ e2eTest: &e2eTest
         command: ./e2e-harness teardown
         when: always
 
-version: 2
+version: 2.1
```

Comment:
Not sure the best place put orbs, and versions.

Reply:
I think it's fine